suspension_data/main.py

```diff
@@ -15,26 +16,17 @@ def read_csv_content() -> tuple[SuspensionRecord]:
 def start():
     records: Final[tuple[SuspensionRecord]] = read_csv_content()
 
-    x_train, y_train = split_data(records)
-    x_test: Final[list[list]] = []
-
-    for year in range(111, 115):
-        x_test.append([
-            Gender.BOY.index,
-            SchoolType.PUBLIC.index,
-            EducationProgram.BACHELORS.index,
-            SuspensionReason.WORK_REQUIREMENTS.index,
-            year
-        ])
-
-    y_test: Final[list] = train_model_and_predict(x_train, y_train, x_test)
+    features, targets = split_data(records)
 
-    # TODO: check the output data is as expected
-    result: list = []
-    for x_data, y_data in zip(x_test, y_test):
-        result = x_data + [y_data]
+    model_data_sources = train_test_split(
+        features,
+        targets,
+        test_size=0.1,
+        random_state=42
+    )
 
-    print(result)
+    model, loss = train_model_and_evaluate(*model_data_sources)
```

Comment:
[Question]
Is the predicting process removed?
I didn't find code related to model prediction.